README.md

```
@@ -29,7 +29,16 @@ I'd love to hear your thoughts! If you have suggestions or find any bugs:
 
 ## Installing
 
-Works via CLI for all major AI coding agents (Cursor, Antigravity, Claude Code, Codex, Windsurf, Copilot, etc.):
+### Claude Code Plugin (Recommended)
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)